app/src/org/commcare/android/framework/SessionActivityRegistration.java

```diff
@@ -52,7 +56,12 @@ public static void handleOrListenForSessionExpiration(Activity activity) {
      * this method in onPause methods of activities that are session sensitive.
      */
     public static void unregisterSessionExpirationReceiver(Activity activity) {
-        activity.unregisterReceiver(userSessionExpiredReceiver);
+        try {
+            activity.unregisterReceiver(userSessionExpiredReceiver);
```

Comment:
Is there any way to do this without  catching the exception? I'm guessing 'no', but generally am a bit squicked by try/catch module rather than checking explicitly